protocols/request-response/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 - Update dependencies.
 
+- Implement `Debug` for `RequestResponseCodec` and `RequestProtocol`. See [PR 2183].
```

Comment:
Is that accurate? The Debug impl of `RequestProtocol` already existed, just now it is hand-rolled.

And I can't see a Debug impl for `RequestResponseCodec` itself.

Is this a leftover from an earlier iteration?

Reply:
Good point. Addressed in https://github.com/libp2p/rust-libp2p/pull/2183/commits/d78576c00a0fa313059d194d55883b931ec8293c.